pyccel/parser/semantic.py

```diff
@@ -830,11 +830,14 @@ def _visit_str(self, expr, **settings):
 
     def _visit_Slice(self, expr, **settings):
         args = list(expr.args)
-        if args[0] is not None:
+        if args[0]:
             args[0] = self._visit(args[0], **settings)
 
-        if args[1] is not None:
+        if args[1]:
             args[1] = self._visit(args[1], **settings)
+
+        if args[2]:
+            args[2] = self._visit(args[2], **settings)
```

Comment:
Can't this be written as the following?
```python
args = [(self._visit(a, **settings) if a else None) for a in expr.args]
```

Reply:
@EmilyBourne It's hard to follow the review process if my comment gets resolved without me knowing how and why. I think the developer should reply to my comment (e.g. "done in commit ...") and then the reviewer should close the conversation...

Reply:
Sorry